src/utils/constants.ts

```diff
@@ -195,6 +195,3 @@ export const GLOBAL_TOKEN_URI_NAME = 'tokenUri';
 export const GLOBAL_BASE_TOKEN_URI_NAME = 'baseTokenUri';
 
 export const ASSET_ID_PREFIX = 'modlpy/pallet_asset';
-
-// TODO @prashantasdeveloper remove this when removing dual version support
-export const BTREE_SET_PORTFOLIO_ID_SPEC_VERSION = 7000004;
```

Comment:
staging and testnet are already on 7000005, so it should be enough to distinguish between v6 and v7 now